packages/multichain/src/scope/constants.ts

```diff
@@ -0,0 +1,56 @@
+import MetaMaskOpenRPCDocument from '@metamask/api-specs';
+
+import type { NonWalletKnownCaipNamespace } from './types';
+
+/**
+ * ScopeStrings for offchain methods that are not specific to a chainId but are specific to a CAIP namespace.
+ */
+export enum KnownWalletScopeString {
+  Eip155 = 'wallet:eip155',
+}
+
+/**
+ * Methods that do not belong exclusively to any CAIP namespace.
+ */
+export const KnownWalletRpcMethods: string[] = [
+  'wallet_registerOnboarding',
+  'wallet_scanQRCode',
+];
+
+const WalletEip155Methods = ['wallet_addEthereumChain'];
+
+/**
+ * All MetaMask methods, except for ones we have specified in the constants above.
+ */
+const Eip155Methods = MetaMaskOpenRPCDocument.methods
+  .map(({ name }: { name: string }) => name)
+  .filter((method: string) => !WalletEip155Methods.includes(method))
+  .filter((method: string) => !KnownWalletRpcMethods.includes(method));
+
+/**
+ * Methods for ecosystem that are chain specific.
```

Comment:
Nit: Perhaps this wording is clearer? Something seemed grammatically wrong here
```suggestion
 * Methods by ecosystem that are chain specific.
```